tools/webpack/shared.js

```diff
@@ -41,6 +41,26 @@ const getBaseConfig = ( env ) => {
 		watch: env.watch,
 		plugins: [
 			new DefinePlugin( {
+				/*
+				 * These variables are part of https://github.com/WordPress/gutenberg/pull/61486
+				 * They're expected to be released in an upcoming version of Gutenberg.
+				 *
+				 * Defining this before the packages are released is harmless.
+				 *
+				 * @todo Remove the non-globalThis defines here when packages have been upgraded to the globalThis versions.
+				 */
+
+				// Inject the `IS_GUTENBERG_PLUGIN` global, used for feature flagging.
+				'globalThis.IS_GUTENBERG_PLUGIN': JSON.stringify(
+					Boolean( process.env.npm_package_config_IS_GUTENBERG_PLUGIN )
```

Comment:
This should be just false. no need to use process.env (same for the next one, it should be true)

Reply:
Fixed, thanks for catching that.